SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialTraceUniqueId.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Interop/Connection/SpatialTraceUniqueId.h"
+
+#include "Utils/SchemaUtils.h"
+
+using namespace SpatialGDK;
+
+namespace
+{
+void ToHex(uint8 N, TCHAR OutHex[3])
+{
+	const TCHAR* Chars = TEXT("0123456789ABCDEF");
+	OutHex[0] = Chars[(N >> 4) & 0xF];
```

Comment:
Perhaps static const this for the class?

Reply:
Ohhhh just saw the comment

Reply:
Yeah could do 👍